mods/gen/credits.txt

```diff
@@ -3,10 +3,12 @@ Main Developer:
 
 Additional Coding:
 	* dnqbob
-    * forcecore/boolbada
+	* forcecore/boolbada
 	* Holloweye
 	* Gyula Zimmermann (Graion Dilach)
 	* Martin Lang (Gerseras/Seraphinexxx)
+	* Developers of Combimed Arms
+	* Developers of OpenHV
```

Comment:
I'm not sure about crediting people like this, if you took code from those mods, i think would be better if you make sure whose code it is and credit them specifically. There's like 4 people i can see that those 2 entries can mean, Matt, Dzierzan, Inq and Darkademic.

Reply:
fixed